components/tenant-mgt-common/org.wso2.carbon.tenant.common/src/main/java/org/wso2/carbon/stratos/common/constants/TenantConstants.java

```diff
@@ -56,7 +56,10 @@ public enum ErrorMessage {
         ERROR_CODE_TENANT_DOES_NOT_MATCH_REGEX_PATTERN("TM-60017", "Invalid tenant domain: %s. " +
                 "Domain should match the regex pattern %s."),
         ERROR_CODE_PRE_TENANT_CREATION_FAILED("TM-60018", "Error occurred in tenant pre creation."),
-        ERROR_CODE_TENANT_LIMIT_REACHED("TM-60019", "Maximum tenant limit reached.");
+        ERROR_CODE_TENANT_LIMIT_REACHED("TM-60019", "Maximum tenant limit reached."),
+        ERROR_CODE_OWNER_NOT_FOUND("TM-60020", "Tenant owner cannot be found for the provided id: %s."),
```

Comment:
Tenant owner cannot be found for the provided tenant id ??

Reply:
Fixed with https://github.com/wso2/carbon-commons/pull/496/commits/7069774521bbf48eddd908e84dcfb3d1aa8d19df